CHANGELOG.md

```diff
@@ -6,7 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 # [Unreleased](https://github.com/purecloudlabs/genesys-cloud-streaming-client/compare/v15.1.1...HEAD)
 
-# [v15.1.1](https://github.com/purecloudlabs/genesys-cloud-streaming-client/compare/v15.0.4...v15.1.1)
+# [v15.1.2](https://github.com/purecloudlabs/genesys-cloud-streaming-client/compare/v15.0.4...v15.1.1)
```

Comment:
Should this range be `v15.1.1...v15.1.2`?

Reply:
yes